From 0e53a889069109cfdbebab0f79c91349d6abb2f5 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 3 Jan 2022 08:44:55 -0800
Subject: [PATCH] drop upper version reqs

Fixed in upstream, should be part of the next release so this patch
can be dropped.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index a21bc87..3a0b74a 100644
--- a/setup.py
+++ b/setup.py
@@ -99,9 +99,9 @@ setup(
     install_requires=[
         "click>=7.1.2",
         "platformdirs>=2",
-        "tomli>=0.2.6,<2.0.0",
+        "tomli>=0.2.6",
         "typed-ast>=1.4.2; python_version < '3.8' and implementation_name == 'cpython'",
-        "pathspec>=0.9.0, <1",
+        "pathspec>=0.9.0",
         "dataclasses>=0.6; python_version < '3.7'",
         "typing_extensions>=3.10.0.0",
         # 3.10.0.1 is broken on at least Python 3.10,
-- 
2.34.1

